docs/reference/query-dsl/semantic-query.asciidoc

```diff
@@ -40,9 +40,209 @@ The `semantic_text` field to perform the query on.
 (Required, string)
 The query text to be searched for on the field.
 
+`inner_hits`::
+(Optional, object)
+Retrieves the specific passages that match the query.
+See <<semantic-query-passage-ranking, Passage ranking with the `semantic` query>> for more information.
++
+.Properties of `inner_hits`
+[%collapsible%open]
+====
+`from`::
+(Optional, integer)
+The offset from the first matching passage to fetch.
+Used to paginate through the passages.
+Defaults to `0`.
+
+`size`::
+(Optional, integer)
+The maximum number of matching passages to return.
+Defaults to `3`.
+====
 
 Refer to <<semantic-search-semantic-text,this tutorial>> to learn more about semantic search using `semantic_text` and `semantic` query.
 
+[discrete]
+[[semantic-query-passage-ranking]]
+==== Passage ranking with the `semantic` query
+The `inner_hits` parameter can be used for _passage ranking_, which allows you to determine which passages in the document best match the query.
```

Comment:
Would this have been a better place to link to `<<semantic-query-passage-ranking>>`?

Reply:
Not sure what you mean here? We usually link to section headings.

Reply:
I think I understand now, but I don't see the point of that. This is the first line of the passage ranking tutorial, so linking to the tutorial here would effectively be a link to itself.

Or do I still misunderstand?

Reply:
🤦 You're right, I thought the link above was to a different page